src/extensions/FlagCommentExtension.php

```diff
@@ -108,7 +130,7 @@ public function doFlag()
 		try {
 			$this->owner->write();
 		} catch (ValidationException $e) {
```

Comment:
is this meant to be \Exception ? instead of ValidationException, if it ValidationException then add the namespace

Reply:
Meant to be ValidationException and namespace is there